app/src/pages/inside/filtersPage/filterGrid/shareFilter/shareFilter.scss

```diff
@@ -0,0 +1,34 @@
+.shared-icon {
+  width: 20px;
+  height: 20px;
+  vertical-align: top;
+  display: inline-block;
+  background-image: url(./img/checkmark-icon.svg);
```

Comment:
No need to store all variations of checkmark icon for different sates. Create one checkmark-icon-inline.svg, parse it via Parser and change it's fill property via styles where it needed